_layouts/services.html

```diff
@@ -24,100 +44,116 @@
     z-index: 11;
   }
 
-  main {
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    text-align: left;
-    padding: 0;
+  .header {
+    width: 100%;
   }
-
-  .header-overlay,
-  .header-image {
-    height: 24em;
+  .header__row {
+    display: flex;
     width: 100%;
+    justify-content: center;
+    padding: 0 2em;
   }
-  .header-overlay {
+  .header__image {
+    object-fit: cover;
     position: absolute;
     overflow: hidden;
-    background-color: #ecf0f1;
-    background-image: linear-gradient(#ecf0f12c, #ecf0f1);
-    opacity: 0.8;
-  }
-  .header-image {
-    object-fit: cover;
-    -webkit-filter: blur(2px);
+    height: 24em;
+    width: 100%;
     filter: blur(2px);
     opacity: 0.7;
     mask-image: linear-gradient(
       to top,
       rgba(0, 0, 0, 0) 0%,
       rgba(0, 0, 0, 0.6) 100%
     );
-    -webkit-mask-image: linear-gradient(
-      to top,
-      rgba(0, 0, 0, 0) 0%,
-      rgba(0, 0, 0, 0.6) 100%
-    );
   }
-  main h1 {
-    text-align: left;
+  .header__title {
+    text-align: center;
     margin-top: 1.5em;
     margin-left: 25px;
     margin-right: 25px;
     font-size: 4em;
     font-weight: bold;
     z-index: 1;
   }
-
-  img.service-logo {
+  .header__service-logo {
     margin-top: 6em;
     margin-left: 25px;
     margin-right: 25px;
     height: 5em;
     object-fit: contain;
     z-index: 1;
   }
+  .header__icon {
+    max-height: 300px;
+    max-width: 600px;
+    padding-right: 2em;
+    z-index: 1;
+  }
+  .header__text {
+    align-content: center;
+    max-width: 800px;
+    z-index: 1;
+  }
+  @media (max-width: 1100px) {
+    .header__row {
+      flex-direction: column;
+      align-items: center;
+    }
+    .header__icon {
+      padding-right: 0;
+      max-height: 200px;
+      max-width: 400px;
+    }
+  }
+  @media (max-width: 950px) {
+    .header__image {
+      height: 22em;
+    }
+    .header__title {
+      font-size: 3.5em;
+      margin-top: 1.5em;
+      margin-right: 1em;
+      margin-left: 1em;
+    }
+    .header__service-logo {
+      height: 4em;
+      margin-right: 1em;
+      margin-left: 1em;
+    }
+    .header__icon {
+      max-width: 400px;
+    }
+  }
+  @media (max-width: 600px) {
+    .header__title {
+      font-size: 2.5em;
+    }
+    .header__service-logo {
+      margin-top: 5em;
+      height: 3em;
+    }
+    .header__icon {
+      max-width: 200px;
+    }
+  }
 
-  .flex-row {
+  .content-row {
     display: flex;
     width: 100%;
     justify-content: center;
-    margin-top: 1.5em;
     padding: 4em;
-  }
-  .flex-row.top {
-    margin-top: 0;
     padding: 0;
```

Comment:
Duplicated `padding` styling. Padding: rem is overruled here so it can be deleted.

Reply:
Well spotted, fixed